src/vs/workbench/contrib/debug/browser/debugService.ts

```diff
@@ -878,7 +878,7 @@ export class DebugService implements IDebugService {
 		const actions = errorActions.filter((action) => action.id.endsWith('.command')).length > 0 ?
 			errorActions :
 			[...errorActions, ...(promptLaunchJson ? [configureAction] : [])];
-		const { choice } = await this.dialogService.show(severity.Error, message, actions.map(a => a.label).concat(nls.localize('cancel', "Cancel")), { cancelId: actions.length });
+		const { choice } = await this.dialogService.show(severity.Error, message, actions.map(a => a.label).concat(nls.localize('cancel', "Cancel")), { cancelId: actions.length - 1 });
```

Comment:
@roblourens @connor4312 fyi, the cancel button was never assigned properly

Reply:
👍 thanks for the fix